attributes/default.rb

```diff
@@ -7,6 +7,8 @@
 # Init system to use
 default['mesos']['init']    = case node['platform']
                               when 'debian' then 'sysvinit_debian'
+                              when 'centos', 'amazon', 'scientific'
+                                node['platform_version'].to_i >= 7 ? 'systemd' : 'upstart'
```

Comment:
OK cool. I didn't see the version mentioned in metadata. What are the plans around the next release of the cookbook with this fix for CentOS/RHEL 7 support?